renderlib/MoveTool.cpp

```diff
@@ -83,6 +83,9 @@ MoveTool::action(SceneView& scene, Gesture& gesture)
     glm::vec3 l1 = normalize(xfmVector(camFrame, glm::vec3(click1.x, click1.y, -1.0)));
 
     LinearSpace3f ref; //< motion in reference space (world for now)
+    if (m_localSpace) {
+      ref = origins.currentReference(scene).l;
```

Comment:
Is LinearSpace3f guaranteed to have normalized vectors?

Reply:
for the moment yes.  There isn't any enforcement inside of LinSpace3f but every usage so far is there for rotations which preserve orthogonality and length of basis vectors. 
If `origins` ever contains a collection of objects instead of a single one, then there may be some more work to do to make sure the ensemble has the right center and rotation.